onnxruntime/core/providers/openvino/backends/basic_backend.h

```diff
@@ -22,16 +28,63 @@ class BasicBackend : public IBackend {
   void Infer(Ort::CustomOpApi& ort, OrtKernelContext* context) override;
 
  private:
-  void StartAsyncInference(Ort::CustomOpApi& ort, OrtKernelContext* context);
+  void StartAsyncInference(Ort::CustomOpApi& ort, OrtKernelContext* context, std::shared_ptr<InferenceEngine::InferRequest> infer_request);
 
-  void CompleteAsyncInference(Ort::CustomOpApi& ort, OrtKernelContext* context);
+  void CompleteAsyncInference(Ort::CustomOpApi& ort, OrtKernelContext* context, std::shared_ptr<InferenceEngine::InferRequest> infer_request);
 
   GlobalContext& global_context_;
   SubGraphContext subgraph_context_;
   mutable std::mutex compute_lock_;
   std::shared_ptr<InferenceEngine::CNNNetwork> ie_cnn_network_;
   std::map<std::string, std::shared_ptr<ngraph::Node>> const_outputs_map_;
+  std::unique_ptr<InferRequestsQueue> inferRequestsQueue_;
+};
+
+class InferRequestsQueue {
+public:
+InferRequestsQueue(InferenceEngine::ExecutableNetwork& net, size_t nireq) {
   InferenceEngine::InferRequest::Ptr infer_request_;
```

Comment:
>infer_request_ [](start = 37, length = 14)

change to infer_request since it's a local variable

Reply:
> > infer_request_ [](start = 37, length = 14)
> 
> change to infer_request since it's a local variable

Done